client/state/initial-state.js

```diff
@@ -88,14 +87,17 @@ export function persistOnChange( reduxStore, serializeState = serialize ) {
 }
 
 export default function createReduxStoreFromPersistedInitialState( reduxStoreReady ) {
-	if ( config.isEnabled( 'persist-redux' ) && ! isSupportUserSession() ) {
-		localforage.getItem( 'redux-state' )
-			.then( loadInitialState )
-			.catch( loadInitialStateFailed )
-			.then( persistOnChange )
-			.then( reduxStoreReady );
-	} else {
-		debug( 'persist-redux is not enabled, building state from scratch' );
-		reduxStoreReady( loadInitialState( {} ) );
-	}
+	require.ensure( 'state', () => {
```

Comment:
Another option to consider is as proposed in #9957, splitting reducer to a separate file, so that `state/index.js` and `state/reducer.js` respectively export only a single default export.